src/build.cc

```diff
@@ -656,13 +653,13 @@ void Plan::UnmarkDependents(const Node* node, set<Node*>* dependents) {
 }
 
 void Plan::Dump() const {
-  printf("pending: %d\n", (int)want_.size());
+  printf("pending: %d\n", static_cast<int>(want_.size()));
```

Comment:
Change the print string for size_t instead of casting to int. You're both casting from unsigned to signed, as well as reducing the resolution.